tests/core/test_flow.py

```diff
@@ -447,6 +448,14 @@ def test_add_edge_returns_edge():
     assert added_edge in f.edges
     assert edge in f.edges
 
+def test_add_edge_from_contant():
+    f = Flow(name="test")
+    value = 1
+    c1 = constants.Constant(value)
+    t1 = Task()
+    f.add_edge(upstream_task=c1, downstream_task=t1, key="foo")
+    assert t1 in f.get_tasks()
+    assert f.constants[t1]['foo'] == value
```

Comment:
```suggestion
    assert f.constants[t1]["foo"] == value
```

in order to pass our formatting checks, then LGTM 👍

Reply:
Alternatively, you can run `black .` on the whole codebase which will autoformat everything for you